docker/ui/bin/startup.sh

```diff
@@ -12,4 +12,5 @@ if [ -z "$WF_SERVER" ];
     echo "using Conductor API server from '$WF_SERVER'"
 fi
 
-nohup node server.js 1>&2 > /app/logs/ui.log
\ No newline at end of file
+# Run server.js in the foreground, sending messages to stdout
+node server.js
```

Comment:
Same as server.  Removed `nohup` to run in the foreground and removed redirects to keep messages going to stdout and stderr.